src/loader.js

```diff
@@ -152,6 +166,10 @@ export function pitch(request) {
       this.addContextDependency(dep);
     }, this);
 
+    if (compilation.errors.length > 0) {
+      return callback(compilation.errors[0]);
+    }
```

Comment:
For catching errors, right?

Reply:
Yes. It was moved down passed the dependencies, to ensure those dependenices are always tracked.

Flipping the error state into a success state on the parent compilation, means that it will also update its registered dependencies. (If the parent compilation would fail, it would retain the previous set.)

If we'd drop out of the child compilation because of an error, _before_ the dependencies of the child compilation are moved into the parent, we would 'lose' them. This way, we won't.

Reply:
Would there be a case where multiple errors need to be shown in the errors array?

Reply:
@ScriptedAlchemy 
Not sure actually. The current version of the plugin only reports the first child compilation error as well.